beluga_ros/cmake/ament.cmake

```diff
@@ -22,8 +22,14 @@ find_package(tf2 REQUIRED)
 find_package(tf2_eigen REQUIRED)
 find_package(tf2_geometry_msgs REQUIRED)
 
-ament_python_install_package(${PROJECT_NAME} PACKAGE_DIR
-                             ${PROJECT_SOURCE_DIR}/tools)
+ament_python_install_package(
+  ${PROJECT_NAME}
+  PACKAGE_DIR
```

Comment:
Not sure how you came up with this :100: 

You rock!

Reply:
When working with ROS 2, always look at the [sources](https://github.com/ament/ament_cmake/blob/rolling/ament_cmake_python/cmake/ament_python_install_package.cmake).